### PR TITLE
Replace WalletConnect v1 and Firebase Crashlytics with pod version #5977

### LINKED ISSUE #5977: Replace walletConnect v1, v2 and firebase with pod version, easier to resolve it ( while switching between branches pods don't resets like spm packages do)

### PR DESCRIPTION
**Pull request by oa-s**
Closes #5977

### CODE REVIEW

**Comment by oa-s** [comment]
after #5693

File: AlphaWallet/Common/Services/FirebaseReportService.swift [hunk 1]
```diff
@@ -21,9 +21,9 @@ extension AlphaWallet {
             }
 
             let file = isRunningTests() ? R.file.googleServiceInfoTestsPlist() : R.file.googleServiceInfoPlist()
-            if let options = file.flatMap({ FirebaseOptions(contentsOfFile: $0.path) }) {
+            if let options = file.flatMap({ FIROptions(contentsOfFile: $0.path) }) {
```
Comment (hboon) [inline AlphaWallet/Common/Services/FirebaseReportService.swift:24]: Does the code difference means we are using an older or newer version of Crashlytics?

**Comment by hboon** [comment]
BTW, in the commit message: "crachlytics" -> "crashlytics"

so we can find it when looking in history :)

File: AlphaWallet/Common/Services/FirebaseReportService.swift (hunk 1 above)
Comment (oa-s) [reply AlphaWallet/Common/Services/FirebaseReportService.swift:24]: i am not quite sure, because we import same version, mabe it different for pods.

**Pull request merged**